### PR TITLE
Perf/sso auth

### PR DESCRIPTION
**Pull request by frenautvh**
| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | 
| License          | BSD 3-Clause

## Description

reuse environment user data for sso auth (avoid making an unecessary call to /me on manager startup)

### CODE REVIEW

**Review by cbourgois — DISMISSED** [review]

**Review by Jisay — DISMISSED** [review]

File: packages/components/ng-ovh-sso-auth/src/provider.js [hunk 1]
```diff
@@ -314,6 +314,24 @@ export default function() {
         : $cookies.USERID;
     };
 
+    /**
+     * @ngdoc function
+     * @name login
```
Comment (antleblanc) [inline packages/components/ng-ovh-sso-auth/src/provider.js:319]:
```suggestion
     * @name setLoggedIn
```

File: packages/components/ng-ovh-sso-auth/src/provider.js [hunk 2]
```diff
@@ -314,6 +314,24 @@ export default function() {
...
+     *
+     * @description
+     * Set logged in data if login is done in the app
+     */
+    this.setLoggedIn = function setLoggedIn(userData) {
+      this.user = userData;
+      isLogged = true;
+      if (userData.state === NIC_STATE_ENUM.incomplete && !allowIncompleteNic) {
+        this.goToSignUpPage();
+      }
```
Comment (antleblanc) [inline packages/components/ng-ovh-sso-auth/src/provider.js:330]: **suggestions:** Seems to duplicate logic from `this.login` method just below (L343).
Maybe we can apply some refactoring.

**Review by antleblanc — CHANGES_REQUESTED** [review]
Nice addition! Just found a wrong definition inside the ngDoc comment block (wrong function name).

**Review by antleblanc — DISMISSED** [review]

**Comment by antleblanc** [comment]
Rewriting git commit history.

**Comment by antleblanc** [comment]
Git commit history has been successfully rewritten.

**Review by antleblanc — APPROVED** [review]

**Pull request merged**